service-registry/src/main/java/org/apache/servicecomb/serviceregistry/consumer/MicroserviceVersions.java

```diff
@@ -160,10 +160,10 @@ protected void safeSetInstances(List<MicroserviceInstance> pulledInstances, Stri
       setInstances(pulledInstances, rev);
       validated = true;
     } catch (Throwable e) {
-      LOGGER.error("Failed to setInstances, appId={}, microserviceName={}.",
-          getAppId(),
-          getMicroserviceName(),
-          e);
+      LOGGER.error(String.format("Failed to setInstances, AppId=%s, MicroserviceName=%s.",
```

Comment:
when will lost exception?
by my test, never happened.

Reply:
yes, i will close it.